src/coreclr/interop/platform.h

```diff
@@ -12,15 +12,21 @@
 #define _ASSERTE(x) assert((x))
 #endif
 
+#ifndef ARRAYSIZE
+#define ARRAYSIZE(x)    (sizeof(x)/sizeof(x[0]))
+#endif // !ARRAYSIZE
+
 #ifdef _WIN32
 #include <Windows.h>
+#endif // _WIN32
+
+#if defined(_WIN32) || defined(HOST_UNIX)
 #include <objidl.h> // COM interfaces
```

Comment:
I don't think the `objidl.h` header file exists on non-Windows platforms.

Reply:
I think the CoreCLR build has a stub header that emulates part of it.

Reply:
Oh. If so, that would be nice.

Reply:
Yes, it's under /pal/inc/rt